ElectronClient/app.js

```diff
@@ -1178,7 +1181,8 @@ class Application extends BaseApplication {
 		for (const itemId of ['copy', 'paste', 'cut', 'selectAll', 'bold', 'italic', 'link', 'code', 'insertDateTime', 'commandStartExternalEditing', 'showLocalSearch']) {
 			const menuItem = Menu.getApplicationMenu().getMenuItemById(`edit:${itemId}`);
 			if (!menuItem) continue;
-			menuItem.enabled = !!note && note.markup_language === MarkupToHtml.MARKUP_LANGUAGE_MARKDOWN;
+			const isHtmlNote = !!note && note.markup_language === MarkupToHtml.MARKUP_LANGUAGE_HTML;
+			menuItem.enabled = !isHtmlNote && layout !== 'viewer' && !!note;
```

Comment:
Then i modified the logic for enabling and disabling the `menuItem` with this condition 
```!isHtmlNote && layout !== 'viewer' && !!note```
which causes the same side-effects as this:
```
 if(isHtmlNote || layout === 'viewer' || !note){ 
        menuItem.enabled = false;
 } else {
	menuItem.enabled = true;
 }
```